CHANGES/7149.bugfix

```diff
@@ -0,0 +1,2 @@
+fixed timeouts when writing payload bodies when ``sock_read`` is set.
+-- by :user:`dtrifiro`
```

Comment:
```suggestion
changed ``sock_read`` timeout to start after writing has finished to avoid read timeouts caused by an unfinished write. -- by :user:`dtrifiro`
```